boards/lilygo/ttgo_t8s3/Kconfig.ttgo_t8s3

```diff
@@ -0,0 +1,7 @@
+# Copyright 2024 Lothar Felten <lothar.felten@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_TTGO_T8S3
+	select SOC_ESP32S3_MINI_N8
```

Comment:
If the board has 16MB, please use proper entry here. 
If it has 8MB PSRAM: 	`SOC_ESP32S3_WROOM_N16R8`
If no PSRAM: `SOC_ESP32S3_WROOM_N16`

Reply:
The device uses:
Winbond 25Q128: 16Mbytes flash
apMemory 6404L: 8MBytes PSRAM

Should I replace "SOC_ESP32S3_MINI_N8" by "SOC_ESP32S3_WROOM_N16R8"
and the dts by "espressif/esp32s3/esp32s3_wroom_n16r8.dtsi"
?

Reply:
You have 2 options here:
- use predefined modules such as one configured in `esp32s3_wroom_n16r8.dtsi`
- OR use generic `esp32s3.dtsi` and add the `flash0` and `spiram` nodes into the board .dts

But both options are technically correct, although second is also formally more correct. I think the second option is better as board is based on discrete components and not on module. You can decide.

Reply:
@marekmatej 
I chose to use the predefined modules from the wroom n16r8 board. This board seems to be very similar to it.